vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx

```diff
@@ -64,10 +64,32 @@ module IntegersAndNumbers =
     /// This is a list of all tuples containing all the numbers from 0 to 99 and their squares.
     let sampleTableOfSquares = [ for i in 0 .. 99 -> (i, i*i) ]
 
-    // The next line prints a list that includes tuples, using %A for generic printing.
+    // The next line prints a list that includes tuples, using '%A' for generic printing.
     printfn "The table of squares from 0 to 99 is:\n%A" sampleTableOfSquares
 
 
+/// Values in F# are immutable by default (save for Arrays).  They cannot be changed
```

Comment:
"save for" sounds slightly archaic and might confuse people who only know a little English.  Perhaps just remove the "(save for Arrays)", it seems a bit out of place to have the exclusion in the intro senetence any case